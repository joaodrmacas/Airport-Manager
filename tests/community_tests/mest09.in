a LIS Portugal Lisbon
a GTM UnitedStates Gotham City
l LSI
a JFK UnitedStates Los	Angeles
q
