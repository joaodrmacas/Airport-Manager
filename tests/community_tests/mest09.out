airport LIS
airport GTM
LSI: no such airport ID
airport JFK
